src/main/java/com/smartsheet/api/models/WorkspaceInclusion.java

```diff
@@ -0,0 +1,40 @@
+package com.smartsheet.api.models;
+
+/*
+ * #[license]
+ * Smartsheet SDK for Java
+ * %%
+ * Copyright (C) 2014 Smartsheet
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * %[license]
+ */
+
+/**
+ * Represents specific objects that can be included in some responses.
+ */
+public enum WorkspaceInclusion {
```

Comment:
The thought that I had was that we would create a new Inclusion object for every resource. That way if we do decide to add more values, we can add it to the Inclusion object for that resource. If we have one that is shared and add values, the resources that use that shared Inclusion object may have more options to include than what is available to that resource. Thoughts?